cypress/e2e/home.cy.js

```diff
@@ -1,8 +1,287 @@
-describe.only("empty spec", () => {
-   it.only("passes", () => {
-     cy.visit("/");
-     cy.url().should('be.equal', `${Cypress.config("baseUrl")}/license`);
-     // Temporary wait statement to make sure license page actually loads
-     cy.wait(1000);
-   });
- });
+/**
+ * Visits document root and accepts the license if redirected to the accept
+ * license page.
+ */
+function acceptsLicense() {
+  cy.visit("/");
+  cy.url().then((url) => {
+    if (url != `${Cypress.config("baseUrl")}/license`) {
+      return;
+    }
+    cy.get("#eula-agree").check();
+    cy.get("#sign").click();
+  })
+}
+
+/**
+ * Visits document root and authenticates as an admin if redirected.
+ */
+function authenticatesAsAdmin() {
+  cy.visit("/");
+  cy.url().then((url) => {
+    if (url != `${Cypress.config("baseUrl")}/auth`) {
+      return;
+    }
+    cy.get('#key').type("test-admin-key{enter}");
+  })
+}
+
+/**
+ * Installs module, but it assumes already on the add-on modules tab of setup.
+ */
+function installModule($moduleList, module) {
```

Comment:
Your other function names describes what it does ("acceptSLicence", "AuthenticateSAsAdmin"), but this does not ("installModule".) Why the change?